### PR TITLE
Add getting started

### PR DESCRIPTION
**Pull request by oScape**
See #2.

### CODE REVIEW

File: src/ch01-00-getting-started.md [hunk 1]
```diff
@@ -1 +1,56 @@
-Explain first section
\ No newline at end of file
+# Getting started
+
+Let's start our journey with Bastion. During it we will see the main features, we have a lot to discover. For that we will follow a theme, we will build a small CRUD application.
```
Comment (o0Ignition0o) [inline src/ch01-00-getting-started.md]:
```suggestion
Let's start our journey with Bastion. During it we will see the main features, we have a lot to discover. For that we will follow a theme, we will build a small [CRUD application](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete).
```

File: src/ch01-00-getting-started.md [hunk 2]
```diff
@@ -1 +1,56 @@
...
+
+In this first chapter, we will talking about:
```
Comment (o0Ignition0o) [inline src/ch01-00-getting-started.md]:
```suggestion
In this first chapter, we will talk about:
```

File: src/ch01-00-getting-started.md [hunk 3]
```diff
@@ -1 +1,56 @@
...
+- Installing Bastion on your project
+- Classic and timeless: hello, world!
```
Comment (o0Ignition0o) [inline src/ch01-00-getting-started.md]:
```suggestion
- The Classic and timeless: hello, world!
```

File: src/ch01-00-getting-started.md [hunk 4]
```diff
@@ -1 +1,56 @@
...
+
+## Installation
+
+Bastion is a library which need to be added as a dependency to your project. For that, you just have to add it as follow to your `Cargo.toml`:
+```rs
+bastion = "0.3.5-alpha"
+```
+
```
Comment (o0Ignition0o) [inline src/ch01-00-getting-started.md:17]:
```suggestion
You can also use [cargo-edit](https://github.com/killercup/cargo-edit) and run `cargo add bastion` in a shell.
```

File: src/ch01-00-getting-started.md [hunk 5]
```diff
@@ -1 +1,56 @@
...
+## Hello, world!
+The most classic of all examples and especially the essential hello, world! Only for you, in Bastion.
+```rs
+fn main() {
```
Comment (o0Ignition0o) [inline src/ch01-00-getting-started.md:25]: probably the same nits as https://github.com/bastion-rs/bastion/pull/234 ^^

**Review by o0Ignition0o — CHANGES_REQUESTED** [review]
Looks great, a couple of nits and we can get this merged!

File: src/ch01-00-getting-started.md (hunk 5 above)
Comment (oScape) [reply src/ch01-00-getting-started.md:25]: Yes, I will change this one, when we will merge the PR on bastion-rs.

**Review by o0Ignition0o — APPROVED** [review]
LGTM :shipit:

**Pull request merged**